We install the manpage into section 1, this updates the file to reflect that.

Index: man/shairport-sync.7
--- man/shairport-sync.7.orig
+++ man/shairport-sync.7
@@ -1,4 +1,4 @@
-.TH shairport-sync 7 User Manuals
+.TH shairport-sync 1 User Manuals
 .SH NAME
 shairport-sync \- Synchronised Audio Player for iTunes / AirPlay
 .SH SYNOPSIS
